From 8c15ff58866e38cb8ce4f7b7ad3ecac941f5863d Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Fri, 7 Sep 2018 20:23:23 +1200
Subject: Fix Makefile.PL to work on perl 5.26+/no-dot-inc perls

Perl 5.26+ have @INC without '.' unless PERL_USE_UNSAFE_INC=1

Perl 5.30+ will have no hack to avoid this problem \o/
---
 Makefile.PL | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.PL b/Makefile.PL
index 76b23e1..a3c10b1 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -1,3 +1,4 @@
+use lib '.';
 use inc::Module::Install;
 
 name           'Number-WithError';
-- 
2.17.1

